experiments/AMIP/moist_mpi_earth/Manifest.toml

```diff
@@ -198,10 +198,12 @@ uuid = "5f86816e-8b66-43b2-912e-75384f99de49"
 version = "0.3.2"
 
 [[deps.ClimaCore]]
-deps = ["Adapt", "BlockArrays", "CUDA", "ClimaComms", "CubedSphere", "DataStructures", "DiffEqBase", "DocStringExtensions", "ForwardDiff", "GaussQuadrature", "GilbertCurves", "InteractiveUtils", "IntervalSets", "LinearAlgebra", "RecursiveArrayTools", "RootSolvers", "Rotations", "SparseArrays", "Static", "StaticArrays", "Statistics", "UnPack"]
-git-tree-sha1 = "39b0b723ff68c0a718653aebaa672aee5556c09f"
+deps = ["Adapt", "BlockArrays", "CUDA", "ClimaComms", "CubedSphere", "DataStructures", "DiffEqBase", "DocStringExtensions", "ForwardDiff", "GaussQuadrature", "GilbertCurves", "HDF5", "InteractiveUtils", "IntervalSets", "LinearAlgebra", "PkgVersion", "RecursiveArrayTools", "RootSolvers", "Rotations", "SparseArrays", "Static", "StaticArrays", "Statistics", "UnPack"]
+git-tree-sha1 = "5144eda908a8324bf1fcafcae2a63f0830e4351c"
+repo-rev = "6a2baa1fa6b2422691786986eecd77909c8f136b"
+repo-url = "https://github.com/CliMA/ClimaCore.jl.git"
```

Comment:
Can we use the released version? Why are we cherry-picking a commit here?

Reply:
This is part of #118 There were some dependency issues in another PR, so we decided to hold off with the updates for the next two PRs (#118 should be mergable in the next couple of days :) ).

Reply:
What dependency issues?

Reply:
There was a behavior change for the updated packages, so we wanted to address this after merging in the regridding PR, this PR and the snow PR, as a separate issue.

Reply:
Updates to Julia 1.8 and the current releases of CA, CC and SF done! :)